apps/core0/subsys/kvm/manager.go

```diff
@@ -852,6 +853,10 @@ func (m *kvmManager) mkDomain(seq uint16, params *CreateParams) (*Domain, error)
 		domain.Devices.Filesystems = append(domain.Devices.Filesystems, fs)
 	}
 
+	if params.KVM ==true{
```

Comment:
You need to configure vscode to auto format source code on save this should be `if params.KVM == true {` (notice the spaces)